corehq/apps/userreports/views.py

```diff
@@ -707,7 +707,8 @@ def export_data_source(request, domain, config_id):
     # First row is taken up by headers
     if params.format == Format.XLS and q.count() >= 65535:
         keyword_params = dict(**request.GET)
-        keyword_params.update(format=Format.CSV)
+        # use default format
+        del keyword_params['format']
```

Comment:
not sure if it's possible but does this ever not exist in `keyword_params`? i think it would crash if it didn't right?